internal/controller/backend_security_policy_test.go

```diff
@@ -241,39 +235,14 @@ func TestBackendSecurityController_RotateCredentials(t *testing.T) {
 	require.NoError(t, cl.Update(t.Context(), awsSecret1))
 
 	// rotate credential
-	_, err = c.rotateCredential(ctx, bsp, oidc, rotator)
+	_, err = rotator.Rotate(ctx)
 	require.NoError(t, err)
 	awsSecret2, err := rotators.LookupSecret(t.Context(), cl, bspNamespace, awsSecretName)
 	require.NoError(t, err)
 	t2 := awsSecret2.Annotations[rotators.ExpirationTimeAnnotationKey]
 	require.NotEqual(t, t1, t2)
 }
 
-func TestBackendSecurityPolicyController_RotateExpiredCredential(t *testing.T) {
```

Comment:
discussed with @aabchoo this test case is no longer needed as we now get the expiration time from the token directly